Patches/Nyaron/Ammo/Ammo_Nyaron.xml

```diff
@@ -0,0 +1,407 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Nyaron race</li></mods>
+			
+		<match Class="PatchOperationSequence">

```

Comment:
Double nested Patch Op Sequence?? Clean this up a bit, please. You only really need the single FindMod op with a `<match Class="PatchOperationAdd">` and add both ammo sets at once.